### PR TITLE
HW-52629, HW-52630

### PR DESCRIPTION
**Pull request by oleksii-lubianyi-epam**

### CODE REVIEW

**Comment by oleksii-lubianyi-epam** [comment]
NB: Should be merged together with https://github.com/hyperwallet/hyperwallet-ios-ui-sdk/pull/26

File: Sources/Model/HyperwalletBankAccount.swift [hunk 1]
```diff
@@ -118,6 +192,258 @@ public final class HyperwalletBankAccount: HyperwalletTransferMethod {
...
+            storage[TransferMethodField.governmentId.rawValue] = AnyCodable(value: governmentId)
+            return self
+        }
+
+        /// Sets the bank account holder's government ID type.
+        ///
+        /// - Parameter governmentIdType: The bank account holder's government ID type.
+        /// - Returns: a self `HyperwalletBankAccount.Builder` instance.
+        public func governmentIdType(_ governmentIdType: GovernmentIdType) -> Builder {
+            storage[TransferMethodField.governmentIdType.rawValue] = AnyCodable(value: governmentIdType)
```
Comment (syyang-hw) [inline Sources/Model/HyperwalletBankAccount.swift]: should be ```governmentIdType.rawValue```

File: Sources/Model/HyperwalletBankAccount.swift [hunk 2]
```diff
@@ -118,6 +192,258 @@ public final class HyperwalletBankAccount: HyperwalletTransferMethod {
...
+            storage[TransferMethodField.firstName.rawValue] = AnyCodable(value: firstName)
+            return self
+        }
+
+        /// Sets the bank account holder's gender.
+        ///
+        /// - Parameter gender: The bank account holder's gender.
+        /// - Returns: a self `HyperwalletBankAccount.Builder` instance.
+        public func gender(_ gender: Gender) -> Builder {
+            storage[TransferMethodField.gender.rawValue] = AnyCodable(value: gender)
```
Comment (syyang-hw) [inline Sources/Model/HyperwalletBankAccount.swift]: should be ```gender.rawValue```

**Comment by oleksii-lubianyi-epam** [comment]
Wrong branch name, closing

**Pull request closed without merging**